### PR TITLE
Added Flowerboy section and changed the functionality of the button. …

### PR DESCRIPTION
**Pull request by Jaxobc**
…I have finaled the last section I will add in my next pull request.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -21,7 +20,30 @@ <h2>About Me</h2>
...
+            The massive bees show that this is not realistic and verify the attractiveness of the Flowerboy. 
+            I imagine that Tyler gave us a glimpse into his extraordinary imagination without allowing us 
+            to loose sense of who his calm character. The music continues to build the narrative he portrayed
+            on the cover by by mixing a variety of songs which blend to share Tyler's current views. He is
+            somehow composed on this album whilst discussing his depression, lonliness and love stories. 
+            The legitimacy of his sentiment is conveyed in his unconventional chord progressions and ertaic
+            developments. By not allowing us to know what comes next Tyler makes us experience his unstelled 
+            feelings. The style is uniquely his own with inspiration drawn from the many artists he features. 
+            This album remains amongst my favourite listens and I could play See You Again on repeat for 
+            hours. I like to think that I could be described as a Flowerboy, both expressive and calm </p>
```
Comment (kennzoid) [inline portfolio/src/main/webapp/index.html:44]: nit: keep indentation consistent

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -21,7 +20,30 @@ <h2>About Me</h2>
...
+            to loose sense of who his calm character. The music continues to build the narrative he portrayed
+            on the cover by by mixing a variety of songs which blend to share Tyler's current views. He is
+            somehow composed on this album whilst discussing his depression, lonliness and love stories. 
+            The legitimacy of his sentiment is conveyed in his unconventional chord progressions and ertaic
+            developments. By not allowing us to know what comes next Tyler makes us experience his unstelled 
+            feelings. The style is uniquely his own with inspiration drawn from the many artists he features. 
+            This album remains amongst my favourite listens and I could play See You Again on repeat for 
+            hours. I like to think that I could be described as a Flowerboy, both expressive and calm </p> 
+        <div id="center"> 
+            <img src="images/tyler.jpg" width="700" height="700" alt="Flowerboy Album Cover"> </div>
```
Comment (kennzoid) [inline portfolio/src/main/webapp/index.html:46]: nit: remove extra space before </div>

Comment (wangtong-g) [reply portfolio/src/main/webapp/index.html:46]: Also, move the closing block element (</div>) to next line.

Here is the suggested formatting rule: https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,14 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomPlayer() {
```
Comment (wangtong-g) [inline portfolio/src/main/webapp/script.js:30]: It's a good habit to add a docstring describing this function.

Comment (wangtong-g) [reply portfolio/src/main/webapp/script.js:30]: Here is the javascript style guide: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

**Pull request merged**